src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const arrayItems: number[] = getNumbers(1, totalPages);
+
+  const handlePageChange = (newPage: number) => {
+    if (currentPage !== newPage) {
+      onPageChange(newPage);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={cn('page-item',
+          { disabled: currentPage === 1 })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href={currentPage !== 1 ? '#prev' : undefined}
+            aria-disabled={currentPage === 1}
+            onClick={currentPage !== 1
```

Comment:
I think no need to add a condition here since we've already checked in the function.  I'm not sure about passing tests though but try to avoid condition here